navigation-timing/nav2_test_redirect_chain_xserver_partial_opt_in.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Navigation Timing 2 WPT</title>
+        <link rel="author" title="Google" href="http://www.google.com/" />
+        <link rel="help" href="http://www.w3.org/TR/navigation-timing-2/#sec-PerformanceNavigationTiming"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="/common/utils.js"></script>
+        <script>
+
+            function verifyTimingEventOrder(eventOrder, timingEntry) {
+                for (var i = 0; i < eventOrder.length - 1; i++) {
```

Comment:
nit here and throughout: use let or const instead of var

Reply:
Yeah, this was copied from an existing test. I'll change it